ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -770,7 +772,7 @@ private class PostgresLedgerDao(
 
   override def lookupTransaction(
       transactionId: TransactionId): Future[Option[(LedgerOffset, LedgerEntry.Transaction)]] = {
-    dbDispatcher.executeSql { implicit conn =>
+    dbDispatcher.executeSql(s"lookup transaction [$transactionId]") { implicit conn =>
```

Comment:
How is a `TransactionId` printed? If it's a tagged type, would it be better to unwrap it for readability?

Reply:
The `toString` is the same as the underlying value:
```
scala> First(3)
res0: Int @@ scalaz.Tags.First = 3

scala> res0.toString
res1: String = 3
```